Sources/LiveViewNative/BuiltinRegistry.swift

```diff
@@ -122,4 +123,58 @@ struct BuiltinRegistry: BuiltinRegistryProtocol {
             try TintModifier(from: decoder)
         }
     }
+    
+    @ViewBuilder
+    static func applyBinding<R: CustomRegistry>(
+        _ binding: AttributeName,
+        event: String,
+        value: Payload,
+        to view: some View,
+        element: ElementNode,
+        context: LiveContext<R>
+    ) -> some View {
+        switch binding {
+        case "phx-window-focus":
+            ScenePhaseObserver(content: view) { scenePhase in
+                if scenePhase == .active {
+                    Task {
+                        try await context.coordinator.pushEvent(type: "focus", event: event, value: value)
+                    }
+                }
+            }
+        case "phx-window-blur":
+            ScenePhaseObserver(content: view) { scenePhase in
+                if scenePhase == .background {
+                    Task {
+                        try await context.coordinator.pushEvent(type: "blur", event: event, value: value)
+                    }
+                }
+            }
+        case "phx-click":
+            // Special case for `Button`, which handles this event itself.
+            if element.tag == "button" {
```

Comment:
I'm not sure if there should be a more general purpose way of representing that an element handles the event internally. Is this rare enough to just handle the few edge cases?